platform/error/platform-error-impl/src/main/resources/codeMessages.properties

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Codice Foundation
+#
+# This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+# version 3 of the License, or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+# <http://www.gnu.org/licenses/lgpl.html>.
+
+403=You are not authorized to see the resource you are attempting to view. Please \
```

Comment:
📓 Using this resource bundle for message lookups was significantly simpler and cleaner than creating a map and managing it in code. It would also, should it ever matter, bring us incrementally closer to a system that could support i18n/l10n.